src/files/BrsFile.ts

```diff
@@ -105,6 +108,8 @@ export class BrsFile implements BscFile {
 
     public program: Program;
 
+    public cachedLookups: CachedLookups;
```

Comment:
Don't forget to mark this as `private` so that plugins don't find this as a replacement for `.parser.references`